todo_app/app.py

```diff
@@ -5,35 +5,47 @@
 from todo_app.data.session_items import add_item
 from todo_app.data.session_items import get_item
 from todo_app.data.session_items import save_item
```

Comment:
These imports break your app since you (correctly) removed the session_items.py file. Delete anything that mentions session_items